### ISSUE TITLE
A11y_Fluent UI_Accordion_Keyboard: Keyboard focus is not visible to the theme button and web light menu item  

### ISSUE THREAD

**Issue by Latha2564** [microsoft/vscode-azurefunctions#3404]
**GitHubTags:**

 #A11ySev2; #TCS-E+d; #NFT-Fluentui-Web-Oct2022; #DesktopWeb; #WCAG2.4.7.
 #Keyboard; #A11yMAS.

**OS Details:** 

 Web:win11

**Environment Details:**

 Application Name: Fluent Ui

**Repro Steps:**

1.Hit enter on the URL https://react-dev.fluentui.dev/
2."Fluent UI React v9" page will open.
3.Tab till "Components" button and hit enter.
4.Tab till "Accordion" link on the left side bar and hit enter. 
5.Tab till "Theme" button and hit enter.
6.Observe keyboard focus is not visible to the theme button and web light menu item. 

**Actual Result:**
  
Keyboard focus is not visible to the theme button and web light menu item.  

**Expected Result:**

Keyboard focus should be visible to the theme button and web light menu item. 

**User Impact:**

Keyboard users are not able to access the theme button and web light menu item if the focus is not visible.

![3404 A11y_Fluent UI_Accordion_Keyboard](https://user-images.githubusercontent.com/89508906/198568581-ffcaff42-6fcf-4f3c-82ca-9090e6cdda01.gif)

**Comment by Latha2564** [comment]
Closing this bug as we logged in incorrect ADO
